src/GitParamTabExpansion.ps1

```diff
@@ -173,6 +177,14 @@ $gitParamValues = @{
     rebase = @{
         strategy = 'resolve recursive octopus ours subtree'
     }
+    restore = @{
+        conflict = 'merge diff3'
+        source = {
+            param($ref)
+            gitBranches $matches['ref'] $true
```

Comment:
I don't always review code, but when I do it's six months later. 💪 

Did you mean to use `$ref` here?

Reply:
Yup, good catch.  I'll submit a PR shortly.